quality-gate.sh

```diff
@@ -25,7 +25,7 @@ verify-cert () {
 }
 
 mli-files () {
-  N=522
```

Comment:
I will not accept a change that increases this number. Create emply mli files for executables instead, and non-empty ones for libraries